allofplos/makedb.py

```diff
@@ -17,14 +18,15 @@
     DateTimeField, BooleanField, IntegerField, IntegrityError
 from playhouse.sqlite_ext import SqliteExtDatabase
 
+from .corpus import Corpus
 from .transformations import filename_to_doi, convert_country
+from . import get_corpus_dir
```

Comment:
you can import starterdir from . too

Reply:
I actually don't think you need to import `get_corpus_dir` or set a `corpusdir` value because the Corpus class already calls it